pandas/core/series.py

```diff
@@ -2378,7 +2378,8 @@ def take(self, indices, axis=0, convert=True, is_copy=False, **kwargs):
         --------
         numpy.ndarray.take
         """
-        nv.validate_take(tuple(), kwargs)
+        if kwargs:
```

Comment:
We can leave it in even then (for performance sensitive methods like take here). It is still 10x faster even after #15850 (although 10x is not that important anymore at those low numbers).